mkdocs.yml

```diff
@@ -7,7 +7,7 @@ repo_url: https://github.com/cozy/cozy-docs-v3
 edit_uri: edit/master/src/
 site_description: Cozy documentation
 site_author: CozyCloud - https://cozy.io
-extra_css: [extra.css]
+extra_css: [css/extra.css, css/highlight_theme.css]
```

Comment:
Could you please do the same for `mkdocs_fr.yml`, which is used to build the french flavor ?

Reply:
Sure. It's done 👍